docs/Makefile

```diff
@@ -1,7 +1,8 @@
 PREFIX := /usr/local
 DATADIR := ${PREFIX}/share
 MANDIR := $(DATADIR)/man
-GOMD2MAN ?= $(shell command -v go-md2man || echo '$(GOBIN)/go-md2man')
+# Following go-md2man is guaranteed on host
+GOMD2MAN ?= ../tests/tools/build/go-md2man
```

Comment:
I'm not seeing the `build` directory in upstream.  Is this getting built somewhere?  Should we instead use `./tests/tools/vendor/github.com/cpuguy83/go-md2man`  or maybe add the two directories to the old command as or's?

Reply:
@TomSweeneyRedHat its generated by the make